txbridge/src/main/java/org/jboss/jbossts/txbridge/outbound/BridgeXAResource.java

```diff
@@ -70,7 +71,7 @@
      * @param externalTxId the parent JTA transaction identifier.
      * @param bridgeWrapper the control for the subordinate WS-AT transaction.
      */
-    public BridgeXAResource(Uid externalTxId, BridgeWrapper bridgeWrapper)
+    public BridgeXAResource(Transaction externalTxId, BridgeWrapper bridgeWrapper)
```

Comment:
@tomjenkinson I don't understand exactly what you mean by two param constructor. I can add there the old constructor
```
public BridgeXAResource(Uid externalTxId, BridgeWrapper bridgeWrapper)
```
but still I would need the `Transaction externalTxId` which is not provided. It's the same with one-arg constructor. What I miss here?